### DISCUSSION TITLE
碰撞检测之线段检测 - Busyo's Blog

### DISCUSSION THREAD

**Discussion by busyoGG** [busyoGG/busyoGG.github.io#63] [Announcements]
https://busyo.buzz/article/ae332e92ff44/ 

 线段检测原理 - Busyo - Busyo's Blog

**Comment by SDU-Rain** [comment]
线段平行的情况下，应该是要用短的线段两端去计算到长的线段的距离，要不然左图那种情况计算出来的结果就不对了~

**Comment by busyoGG** [comment]
> @SDU-Rain
> 线段平行的情况下，应该是要用短的线段两端去计算到长的线段的距离，要不然左图那种情况计算出来的结果就不对了~

确实会有这种情况，感谢你的指正👍
